library/src/androidTest/java/in/krharsh17/barview/ExampleInstrumentedTest.java

```diff
@@ -8,6 +8,8 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
+import static org.junit.Assert.assertEquals;
+
```

Comment:
Is this necessary?

Reply:
you just used assertEquals and didn't imported it, so, it was showing error. that's why I imported it

Reply:
Alright